fabric-loot-tables-v1/src/main/java/net/fabricmc/fabric/api/loot/v1/event/LootTableLoadingCallback.java

```diff
@@ -33,17 +32,17 @@
 public interface LootTableLoadingCallback {
 	@FunctionalInterface
 	interface LootTableSetter {
-		void set(LootTable supplier);
+		void set(LootTable table);
 	}
 
 	Event<LootTableLoadingCallback> EVENT = EventFactory.createArrayBacked(
 			LootTableLoadingCallback.class,
-			(listeners) -> (resourceManager, manager, id, supplier, setter) -> {
+			(listeners) -> (resourceManager, manager, id, table, setter) -> {
 				for (LootTableLoadingCallback callback : listeners) {
-					callback.onLootTableLoading(resourceManager, manager, id, supplier, setter);
+					callback.onLootTableLoading(resourceManager, manager, id, table, setter);
 				}
 			}
 	);
 
-	void onLootTableLoading(ResourceManager resourceManager, LootManager manager, Identifier id, FabricLootSupplierBuilder supplier, LootTableSetter setter);
+	void onLootTableLoading(ResourceManager resourceManager, LootManager lootManager, Identifier id, FabricLootTableBuilder table, LootTableSetter setter);
```

Comment:
Will this break old code compiled against the old version?

Reply:
I shouldn't because FabricLootSupplierBuilder extends FabricLootTableBuilder

Reply:
No, but isn't *compiled* code looking for `onLootTableLoading(ResourceManager,LootManager,Identifier,FabricLootSupplierBuilder,LootTableSetter)`, which has been removed?